### ISSUE TITLE
Wrong fragments appeared after second pass

### ISSUE THREAD

**Issue by sun510001** [NVlabs/nvdiffrec#107]
Hello, Firstly, thanks for your excellent work.
I'm trying nvdiffrec and nvdiffrecmc on my own data. 

It works pretty good at first pass:
![img_dmtet_pass1_000077](https://user-images.githubusercontent.com/32349603/208568722-978914e7-8852-4107-9e82-f2746d2134bc.png)
![img_dmtet_pass1_000042](https://user-images.githubusercontent.com/32349603/208569285-4ef78fc3-d229-474a-ac03-52fcc784f108.png)
![image](https://user-images.githubusercontent.com/32349603/208570411-3a8cf8e8-66aa-4ef8-9fec-daf72384d8c4.png)
![image](https://user-images.githubusercontent.com/32349603/208570390-304e504c-59d9-4725-8d43-69100d9bba19.png)

but lots of wrong fragments are appeared after second pass:
![img_mesh_pass_000077](https://user-images.githubusercontent.com/32349603/208568790-e38fe73b-8483-49ab-a2d3-a63dba5150c7.png)
![img_mesh_pass_000042](https://user-images.githubusercontent.com/32349603/208569297-51df80fd-e61a-4cde-9c6e-f53dbb8de09d.png)
![image](https://user-images.githubusercontent.com/32349603/208570290-030517d0-f96f-4193-8721-f8e73404a206.png)
![image](https://user-images.githubusercontent.com/32349603/208570335-0279a429-b1f9-4ed5-a96c-acc1b08637e0.png)

This problem occurs with both nvdiffrec and nvdiffrecmc.
Lock pos can solve this problem, but I want to use regularizer to make mesh smoother.
I have tried laplace_scale  3000 6000 and 10000; laplace: absolute and relative, but wrong fragments still appeared after second pass.

Any input here would be greatly appreciated, thank you!

**Comment by jmunkberg** [comment]
Thanks @sun510001 !

Increasing the Laplacian regularizer would have been my first tests as well. You can test the 
`"lock_pos" : true` option in the config, which disables geometry optimization in the second pass . 
 You can also test decreasing the learning rate in the second pass, like
`"learning_rate": [0.03, 0.003],` or similar. 

Also, nvdiffrec(mc) is quite sensitive to inaccuracies in the foreground segmentation masks and poses.

**Comment by sun510001** [comment]
@jmunkberg  Thanks for your reply.

I checked the previously generated meshes and datasets. The datasets with good matting generated fewer fragments in the textures of the mesh.
